src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -65,7 +66,19 @@ export default class MxGraphRenderer {
       const bounds = shape.bounds;
       const parent = this.getParent(bpmnElement);
       const absoluteCoordinate = { x: bounds.x, y: bounds.y };
-      this.insertVertexGivenAbsoluteCoordinates(parent, bpmnElement.id, bpmnElement.name, absoluteCoordinate, bounds.width, bounds.height, bpmnElement.kind);
+      let style: string;
+      if (bpmnElement instanceof ShapeBpmnEvent) {
+        console.log('_________', 'ShapeBpmnEvent', bpmnElement);
```

Comment:
:warning: console log should be removed IMHO

Reply:
it is WIP - only that is why it stays here

Reply:
done!